pkg/config/lima_config_applier.go

```diff
@@ -43,6 +44,23 @@ func (lca *limaConfigApplier) Apply() error {
 
 	limaCfg.CPUs = lca.cfg.CPUs
 	limaCfg.Memory = lca.cfg.Memory
+	limaCfg.Mounts = []limayaml.Mount{}
```

Comment:
Won't this statement erase the home directory mount in finch.yaml?

Reply:
No. I mentioned in the design doc. https://github.com/runfinch/finch/blob/1bac92a99f2e031792018dedfb6425488af51f1e/docs/config_to_support_additional_directories.md?plain=1#L43

Reply:
Thanks. Added the doc PR link in the description of this PR.